x-pack/plugins/observability/public/components/slo/slo_status_badge/slo_active_alerts_badge.tsx

```diff
@@ -14,11 +14,12 @@ import { paths } from '../../../../common/locators/paths';
 import { useKibana } from '../../../utils/kibana_react';
 
 export interface Props {
+  viewMode?: 'compact' | 'default';
```

Comment:
Shouldn't this be consistent with what you used here? (cardView | listView)

https://github.com/elastic/kibana/pull/171422/files#diff-45d3a79f61bdc1f5de1b9dbdfa77ef48fa4c32a2ea7b79027ad81419f523b4f1R21

Reply:
Or it looks like you have a compact view just for the active alerts, then I think it makes sense to keep it as you have it.

Reply:
Yes, it's only for this badge